bin/commands/folder/create.js

```diff
@@ -0,0 +1,17 @@
+const helpers = require('../../helpers')
+
+module.exports = {
+  command: 'create <organization-id> <folder-name>',
+  desc: 'Create a folder.',
+  builder: {},
+  handler: async function (argv) {
+    try {
+      const client = helpers.getClient(argv)
+      const result = await client.createFolder(argv.organizationId, argv.folderName)
+      helpers.print(result)
+    } catch (error) {
+      throw error
+    }
+    process.exit(0)
```

Comment:
I haven't gone through and tracked down where these exceptions end up (so maybe you're doing this already), but if this request blows up then I think we should do `process.exit(1)`. We should even consider doing it if the [API returns an error](https://ghostinspector.com/docs/api/errors/) (which I know weirdly have 200 statuses, so we'd have to do some checking of the response).

The idea is that these are often going to be used in CI pipelines and if the command fails, it should blow up.

Reply:
Yeah, if we throw an error `yargs` will do the right thing already:

```
❯ ghost-inspector folder get 1234
ghost-inspector folder get <folder-id>

Fetch a single folder.

Options:
  --version  Show version number                                                                               [boolean]
  --apiKey   Your Ghost Inspector API key.              [required] [default: "b86c4cabce54fac5f6c2cd12db2cc1cd2c4f3672"]
  --help     Show help                                                                                         [boolean]

Error: Some error
    at Object.handler (/Users/magius/workspace/node-ghost-inspector/bin/commands/folder/get.js:8:11)
    at Object.runCommand (/Users/magius/workspace/node-ghost-inspector/node_modules/yargs/lib/command.js:242:26)
    at Object.parseArgs [as _parseArgs] (/Users/magius/workspace/node-ghost-inspector/node_modules/yargs/yargs.js:1087:30)
    at Object.runCommand (/Users/magius/workspace/node-ghost-inspector/node_modules/yargs/lib/command.js:200:45)
    at Object.parseArgs [as _parseArgs] (/Users/magius/workspace/node-ghost-inspector/node_modules/yargs/yargs.js:1087:30)
    at Object.get [as argv] (/Users/magius/workspace/node-ghost-inspector/node_modules/yargs/yargs.js:1021:21)
    at Object.<anonymous> (/Users/magius/workspace/node-ghost-inspector/bin/cli.js:12:3)
    at Module._compile (internal/modules/cjs/loader.js:999:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:32)

node-ghost-inspector git/aaronfay/ch4307/add-cli-commands-to-node-library*
❯ echo $?
1
```